### DISCUSSION TITLE
PokeApi Docker image instructions for use

### DISCUSSION THREAD

**Discussion by ingmarfjolla** [PokeAPI/pokeapi#846] [General]
Hello, I was wondering if anyone had some new directions on how to use the PokeApi Docker image? Going here: https://hub.docker.com/r/pokeapi/how-to-use-this-image leads to a dead page now. Thanks

**Comment by Naramsim** [comment]
Hi, https://hub.docker.com/r/pokeapi/pokeapi

It's meant to be used with our Compose files. You find all the info in the readme
